src/components/search/FacetListFreeAll.jsx

```diff
@@ -11,9 +13,14 @@ const FacetListFreeAll = ({
   items,
   showAllCatalogs,
   setShowAllCatalogs,
+  refinementsFromQueryParams,
 }) => {
+  const history = useHistory();
   const handleInputOnChange = () => {
```

Comment:
When the user selects an option different than the default option for the "Free/All" facet, ideally we'd also be putting that into URL as a query parameter so that we can maintain the user's search history and their selection would persist on refresh.

While we're in here dealing with query parameters for the "Free/All" facet, I think it's worth adding that in to be consistent with the other facets.

Example:
* "Free to me" (default) is selected: no query parameter since its the default selection
* "All courses" is selected: `?free_all=all` or similar

Reply:
This was looking doable in a short time frame, but the way we get refinements vs. query params and clearing the filters is proving a bit tricky. We originally talked about adding this later, so I'm going to open a separate ticket for it.